mix.exs

```diff
@@ -60,6 +60,7 @@ defmodule Media.MixProject do
       {:plug_cowboy, "~> 2.0"},
       {:ex_aws, "~> 2.1"},
       {:ex_aws_s3, "~> 2.0"},
+      {:sweet_xml, ">= 0.0.0"},
```

Comment:
@zaher-hs what is this deps for ?

Reply:
this is used for AWS sending/receiving requests I think it is something related to encoding and decoding the request.